python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -761,3 +837,29 @@ class RandomForestClassifier(Base):
             print_rf_detailed(rf_forest64)
         else:
             print_rf_detailed(rf_forest)
+
+    def get_treelite_forest_from_rf(self, num_features,
```

Comment:
Is this function user callable? If so it needs a docstring, also a better name. It could be `get_treelite` to be honest. Otherwise make it:
```suggestion
    def _get_treelite_forest_from_rf(self, num_features,
```